installer/platform2asic.conf

```diff
@@ -0,0 +1,69 @@
+# List of two main platfrom: broadcom and mellanox
```

Comment:
># List of two main platfrom [](http://example.com/codeflow?start=0&length=27)

I am proposing a general solution in PR [https://github.com/Azure/sonic-buildimage/pull/8542](https://github.com/Azure/sonic-buildimage/pull/8542).